src/anyio/_backends/_asyncio.py

```diff
@@ -1838,8 +1838,6 @@ def _create_task_info(task: asyncio.Task) -> TaskInfo:
 
 
 class TestRunner(abc.TestRunner):
-    _send_stream: MemoryObjectSendStream[tuple[Awaitable[Any], asyncio.Future[Any]]]
```

Comment:
why did you remove this?

Reply:
I didn't write this code, I merged from the contributor's branch.

Reply:
I've just reverted that change.